script/DeployAAVEBorrower.s.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.28;
+
+import "forge-std/Script.sol";
+import {Utils} from "script/libraries/Utils.sol";
+import {EnvReader} from "script/libraries/EnvReader.sol";
+import { ICowSettlement } from "../src/interface/ICowSettlement.sol";
+import { IBorrower } from "../src/interface/IBorrower.sol";
+import { FlashLoanRouter } from "../src/FlashLoanRouter.sol";
+import { AaveBorrower } from "../src/AaveBorrower.sol";
+
+contract DeployAAVEBorrower is Script, EnvReader, Utils {
+    constructor() {
+        // Initialise contracts
+        settlement = addressEnvOrDefault("SETTLEMENT_CONTRACT", DEFAULT_SETTLEMENT_CONTRACT);
```

Comment:
I think this should always be hardcoded (since our settlement contract always has the same address)